src/main/java/frc/robot/subsystems/climb/Climb.java

```diff
@@ -1,5 +1,5 @@
 package frc.robot.subsystems.climb;
-import java.util.TreeMap;
+
```

Comment:
ngl I don't have the bandwidth to look at this right now. you didn't make any changes, just ran the formatter right?